tests/src/test/scala/quasiquotes/ErrorSuite.scala

```diff
@@ -297,9 +297,11 @@ class ErrorSuite extends FunSuite {
       val tpes = List(q"T")
       q"expr: ..$tpes"
     """) === """
-      |<macro>:5: identifier expected but ellipsis found
+      |<macro>:5: type mismatch when unquoting;
+      | found   : List[scala.meta.Term.Name]
+      | required: scala.collection.immutable.Seq[scala.meta.Type]
```

Comment:
I don't know exactly, why this changed, but new error message seems to be informative.
For list of types user'd get `found   : scala.collection.immutable.Seq[scala.meta.internal.ast.Type]
 required: scala.meta.internal.ast.Type
       q"expr: ..$tpes"
                 ^
`